app/lib/actions/remote_execution/run_host_job.rb

```diff
@@ -173,6 +194,10 @@ def determine_proxy!(proxy_selector, provider, host)
         end
         proxy
       end
+
+      class Jail < ::Actions::ObservableAction::Jail
+        allow :host_name, :host_id, :host, :job_invocation_id, :job_invocation
+      end
```

Comment:
Just a crazy idea: do we also want to expose the script itself? Just in case someone want to archive those? On the other hand, these could contain sensitive data, so it's not bad thing to avoid this.

Reply:
> these could contain sensitive data

That's what I thought as well.